tests/libcxx/expected_results.txt

```diff
@@ -566,48 +524,43 @@ std/utilities/allocator.adaptor/allocator.adaptor.members/construct_pair_rvalue.
+std/language.support/support.limits/support.limits.general/memory.version.compile.pass.cpp FAIL
 
 # libc++ doesn't implement P2231R1 Add further constexpr support for optional/variant
-std/language.support/support.limits/support.limits.general/variant.version.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/variant.version.compile.pass.cpp FAIL
 
 # Tests non-portable behavior
 std/utilities/format/format.formatter/format.context/format.context/advance_to.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/arg.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/ctor.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/locale.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/out.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.bool.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.c_string.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.char.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.const_char_array.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.signed_integral.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.string.pass.cpp FAIL
-std/utilities/format/format.formatter/format.context/format.formatter.spec/formatter.unsigned_integral.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.bool.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.char_array.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.const_char_array.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.c_string.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.handle.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.pointer.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.signed_integral.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.string.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.unsigned_integral.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/types.compile.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.char.pass.cpp FAIL
+std/utilities/format/format.formatter/format.formatter.spec/formatter.floating_point.pass.cpp FAIL
 
 # libc++ doesn't yet implement P2216R3 (test uses non-constant-expression format strings)
 std/utilities/format/format.functions/format.pass.cpp FAIL

```

Comment:
libc++ now implements P2216R3. Tests fail because -NaN is formatted differently (and other reasons).

Reply:
Thanks!
I created https://reviews.llvm.org/D131336